index.html

```diff
@@ -6348,6 +6348,41 @@ <h2>Acknowledgements</h2>
 standards process.
     </p>
 
+    <p>
+The Editors gratefully acknowledge the work that led to the creation of this
+specification, and extend a sincere appreciation to those individuals that
+worked on technologies and specifications that deeply influenced this Working
+Group. Namely, the work of Phil Zimmerman, Jon Callas, Lutz Donnerhacke, Hal
+Finney, David Shaw, and Rodney Thayer on <a
+href="https://en.wikipedia.org/wiki/Pretty_Good_Privacy"> Pretty Good Privacy
+(PGP)</a> in the 1990s and 2000s.
+    </p>
+
+    <p>
+In the mid-2010s, preliminary implementations of what would become Decentralized
+Identifiers were <a
+href="https://web-payments.org/minutes/2014-05-07/#topic-1">built</a> in
+collaboration with Jeremie Miller's Telehash project and the W3C Web Payments
+Community Group's work led by Dave Longley and Manu Sporny.
+Some of the <a
+href="https://github.com/WebOfTrustInfo/rwot2-id2020/blob/master/final-documents/requirements-for-dids.pdf">first
+written papers</a> exploring the concept of Decentralized Identifiers can be
```

Comment:
Suggestion to also link to the very early and important DPKI paper. Also, this way we'd have links to papers from both RWoT1 and RWoT2.

```suggestion
Some of the first <a
href="https://github.com/WebOfTrustInfo/rwot1-sf/blob/master/final-documents/dpki.pdf">written</a> <a
href="https://github.com/WebOfTrustInfo/rwot2-id2020/blob/master/final-documents/requirements-for-dids.pdf">papers</a> exploring the concept of Decentralized Identifiers can be
```